### PR TITLE
Update local development guide

### PR DESCRIPTION
**Pull request by saltcod**
Update the local dev guide to add some more opinion and define an ideal workflow via local setup.

### CODE REVIEW

File: apps/docs/pages/guides/cli/local-development.mdx [hunk 1]
```diff
@@ -147,95 +177,195 @@ Database changes are managed through "migrations." Database migrations are a com
...
-```
+        To that file, add the SQL to create this `employees` table
+    </StepHikeCompact.Details>
 
-This creates a new migration named `supabase/migrations/<timestamp>_create_employees.sql`, representing any changes made to the local database since [`supabase start`](/docs/reference/cli/usage#supabase-start).
+    <StepHikeCompact.Code>
+      ```sql
 
-### Add sample data
+create table
```
Comment (saltcod) [inline apps/docs/pages/guides/cli/local-development.mdx:215]: I think we should think about this sql statement as its own "how-to" or reference. 

A user should ideally be able to create a new table from this example really easily. What other data types should we include in here that would be useful for people? 

Should the id primary key be an integer or something else? A uuid?

**Review by Isaiah-Hamilton — APPROVED** [review]

File: apps/docs/pages/guides/cli/local-development.mdx (hunk 1 above)
Comment (saltcod) [reply apps/docs/pages/guides/cli/local-development.mdx:215]: suggestions here: 
an auto-incrementing id (serial?) 
a number
a date

File: apps/docs/pages/guides/cli/local-development.mdx [hunk 2]
```diff
@@ -147,95 +177,217 @@ Database changes are managed through "migrations." Database migrations are a com
...
+insert into
+public.employees (name)
 values
-  ('Erlich Bachman'),
-  ('Richard Hendricks'),
-  ('Monica Hall');
-```
+('Erlich Bachman'),
+('Richard Hendricks'),
+('Monica Hall');
```
[the remark this answers is not in the document]
Comment (saltcod) [reply apps/docs/pages/guides/cli/local-development.mdx:315]: I see a mix of bigint and serial for ids. Any preference there?

File: apps/docs/pages/guides/cli/local-development.mdx [hunk 3]
```diff
@@ -68,12 +84,26 @@ This command may take a while to run if this is the first time using the CLI.
...
+service_role key: eyJh......
 
-You can access services directly with any Postgres client or through the API Gateway ([Kong](https://github.com/Kong/kong)).
+```
+
+You can use the [supabase stop](/docs/reference/cli/usage#supabase-stop) command at any time to stop all services.
+
+## Access your project's services
+
+You can now visit your local Dashboard at [http://localhost:54323](http://localhost:54323), and access the services directly with any Postgres client or through the API Gateway ([Kong](https://github.com/Kong/kong)).
```
[the remark this answers is not in the document]
Comment (saltcod) [reply apps/docs/pages/guides/cli/local-development.mdx:106]: Humm yeah actually not sure why I left that in from the original. I have no idea what it means. 

Think we should expand or remove?

File: apps/docs/pages/guides/cli/local-development.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (saltcod) [reply apps/docs/pages/guides/cli/local-development.mdx:316]: wrong comment there ^^ 

good call on email being not null, I'll adjust

[the remark this answers is not in the document]
Comment (gregnr) [reply apps/docs/pages/guides/cli/local-development.mdx:316]: For `id` column I think we want:
```sql
id bigint primary key generated always as identity
```
Sounds like `serial` and `bigserial` are no longer recommended.

File: apps/docs/pages/guides/cli/local-development.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (gregnr) [reply apps/docs/pages/guides/cli/local-development.mdx:106]:
```suggestion
You can now visit your local Dashboard at [http://localhost:54323](http://localhost:54323), and access the database directly with any Postgres client via `postgresql://postgres:postgres@localhost:54322/postgres`.
```

File: apps/docs/pages/guides/cli/local-development.mdx [hunk 4]
```diff
@@ -147,95 +177,219 @@ Database changes are managed through "migrations." Database migrations are a com
...
+      ```sql
+
+create table
+employees (
+id bigint primary key generated always as identity,
+created_at timestamp with time zone default now(),
+name text,
+email text,
+created_at timestampz default now()
+);
```
Comment (gregnr) [inline apps/docs/pages/guides/cli/local-development.mdx:222]: Heads up, we got `created_at` twice now.

Comment (saltcod) [reply apps/docs/pages/guides/cli/local-development.mdx:222]: oof, I decide to try vscode's new merge editor one single time and get this!

**Pull request merged**